cpu/lpc2387/Makefile.features

```diff
@@ -1 +1,6 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_gpio periph_gpio_irq
```

Comment:
Only this is line is needed. Since the others (rtc, timer) require a specific config at board level, they should be at board level. Ok, for rtc, this is no so obvious but it highly depends on the board setup. Other platforms never provide rtc at cpu level btw.

Reply:
On the LPC2387 the board only needs to specify the number of timers. And actually for SPI no configuration is needed also (except for setting SPI_NUMOF to 1). Maybe it both could be added the CPU in this case?

E.g. the ESP32 also defines `periph_timer` and `periph_rtc` at CPU level, the ESP8266 defines `periph_timer` at CPU level.

Reply:
The RTC also doesn't need config.
A lot of boards define `RTC_NUMOF` but when you grep for it you will find that it is never used anywhere.

I'm preparing a PR…

Reply:
> On the LPC2387 the board only needs to specify the number of timers

So this is a configuration required at board level.

> the ESP32 also defines periph_timer and periph_rtc at CPU level

In general, I don't think we should take this platform as an example for good practices.

Reply:
Done